lib/mcb/courses_editor.rb

```diff
@@ -21,24 +21,46 @@ def initialize(provider:, requester:, course_codes: [])
 
     def run
```

Comment:
"Method run has a Cognitive Complexity of 6 (exceeds 5 allowed)."
I think this reads fine. Ignore.